### DISCUSSION TITLE
Initial Token Allocation Address

### DISCUSSION THREAD

**Discussion by garysdevil** [filecoin-project/community#583] [Q&A]
Referenced to https://spec.filecoin.io/systems/filecoin_token/token_allocation/
Who can fill in the form below?

Total Balance = 2,000,000,000

| Org                      | Proportion | Address     | Balance       |
| ------------------------- | ---------- | ----------- | ------------- |
| Storage Mining Allocation | 55%        | f02         | 1,100,000,000 |
| Mining Reserve            | 15%        | f090        | 276,400,000   |
| Protocol Labs             | 10.5%      | f0117~f0120 | 50,000,000*4  |
| PL Team & Contributor     | 4.5%       |             |               |
| Fundraising - SAFT2007    | 7.5%       |             |               |
| Fileocin Foundation       | 5%         | f0121       | 100,000,000   |
| Fundraising - Remainer    | 2.5%       |             |               |
|                           |            |             |               |

| ID     | Org                       | initial balance | 20230130 balance（million） |
| ------ | ------------------------- | --------------- | --------------------------- |
| f01000 |                           | 20886           | 0                           |
| f01001 |                           | 20886           | 0                           |
| f01002 |                           | 20886           | 0                           |
| f0112  |                           | 200000          | 0                           |
| f0113  |                           | 200000          | 0                           |
| f0114  |                           | 200000          | 0                           |
| f0100  |                           | 979113          | 0                           |
| f0101  |                           | 979113          | 0                           |
| f0102  |                           | 979113          | 0                           |
| f0110  |                           | 10000000        | 0                           |
| f0111  |                           | 10000000        | 0                           |
| f0115  |                           | 50000000        | -30                         |
| f0116  |                           | 50000000        | -30                         |
| f0117  |                           | 50000000        | 30                          |
| f0118  |                           | 50000000        | 30                          |
| f0119  |                           | 50000000        | 30                          |
| f0120  |                           | 50000000        | 30                          |
| f0121  | Fileocin Foundation       | 100000000       | 64                          |
| f0122  |                           | 200000000       | 0                           |
| f090   | Mining Reserve            | 276400000       | 282                         |
| f02    | Storage Mining Allocation | 1100000000      | 842                         |

**Comment by web3jenks** [comment]
@garysdevil how were the confirmed addresses found above? Was it from Lotus code base or configuration? :-)
